frontend/app/auth/login/page.tsx

```diff
@@ -27,21 +28,28 @@ export default function Login() {
     password: '',
   })
 
+  const toast = useToast()
+
   const handleSubmit = async (
     e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
   ) => {
     e.preventDefault()
-    try {
-      await signIn('credentials', {
-        email: formState.email,
-        password: formState.password,
-        redirect: false,
-        callbackUrl: '/',
+
+    const result = await signIn('credentials', {
+      email: formState.email,
+      password: formState.password,
+      redirect: false,
+      callbackUrl: '/',
+    })
+    if (result.error) {
+      toast({
+        title: 'このメールアドレスは登録されていません',
```

Comment:
これってパスワードが違う場合もエラーになりますよね？

だとこの文はおかしいので修正しましょう。

```suggestion
        title: 'このメールアドレスは登録されていないかパスワードが誤っています',
```

豆知識的に知っておいてもらいたいのですが
メールアドレスだけ合っていてパスワードが誤っている場合は
パスワードが誤っていますだけのメッセージは出さないようにしましょう！
→じゃあメールアドレスは合ってるんだ！
って思われて悪用に繋がりかねないので。

このメールアドレスは登録されていないかパスワードが誤っています
みたいに曖昧な表示するのがgoodです！